packages/cli/src/cli.test.ts

```diff
@@ -0,0 +1,275 @@
+import {
+    afterAll,
+    beforeAll,
+    beforeEach,
+    describe,
+    expect,
+    it,
+    vitest,
+} from 'vitest';
+import fs from 'fs';
+
+import { cli } from './cli';
+import { randomUUID } from 'crypto';
+import { assert } from 'console';
+import path from 'path';
+
+describe('CLI tests', () => {
+    describe('`generate` command tests', () => {
+        const url = 'postgres://postgres:postgres@localhost:5432/postgres';
+        const out = 'generated/synthql/tests';
+        const defaultSchema = 'pg_catalog';
+        const schemas = ['public'];
+
+        beforeAll(() => {
+            fs.rmSync(out, {
+                force: true,
+                recursive: true,
+            });
+        });
+
+        beforeEach(() => {
+            vitest.resetModules();
+            vitest.resetAllMocks();
+        });
+
+        it(
+            'Generate schema types with the CLI options default values',
+            async () => {
+                const outDir = `${out}/${Date.now()}-${randomUUID()}`;
+
+                const argv = ['node', 'generate-default.js', 'generate'];
+
+                // Compose file path
+                const outFilePath = path.join(outDir, `db.ts`);
+
+                // Check that file with the name doesn't exist yet
+                assert(!fs.existsSync(outFilePath));
```

Comment:
Instead of using the assert module you should rely on normal expect() assertions.

Reply:
yeah

Reply:
Updated!